django/db/models/sql/compiler.py

```diff
@@ -957,7 +1007,7 @@ def execute_sql(self, result_type):
             if is_empty:
                 rows = aux_rows
                 is_empty = False
-        return rows
+        return (rows, returning_values)
```

Comment:
Yes, that feature request is tracked in https://code.djangoproject.com/ticket/19527.